src/api/app/models/kiwi/image.rb

```diff
@@ -62,6 +63,27 @@ def self.build_from_xml(xml_string, md5)
       new_image.repositories.build(attributes)
       order += 1
     end
+    package_groups = xml["packages"]
+    package_groups = [xml["packages"]] if xml["packages"].is_a?(Hash)
+    package_groups.each do |package_group_xml|
+      attributes = {
+        kind:         package_group_xml['type'],
```

Comment:
You can disable STI by setting this in the model:
```ruby
self.inheritance_column = nil
```
What does everybody think about that? To me it makes more sense to do this but maybe others have different opinions?